packages/frontend/web-ui/src/auth/pages/Login.jsx

```diff
@@ -1,78 +1,116 @@
-                    />
-                </div>
-            </div>
-            <div className="mb-5 row">
-                <label htmlFor="passwordLabel" className="col-sm-2 col-form-label text-start">Password</label>
-                <div className="col-sm-10">
-                    <input 
-                        type="password" 
-                        className="col-sm-10 form-control"
-                        placeholder="********"
-                        name="password"
-                        value={formState.password}
-                        onChange={onInputChange}
-                    />
-                </div>        
-            </div>
-            <div className="row justify-content-evenly">
-                <button onClick={login} className="btn btn-first-cornie mt-2 col-4">Login</button>
-                <button onClick={onResetForm} className="btn btn-second-cornie mt-2 col-4">Delete</button>
-            </div>
-        </>
+
+      <AuthLayout title="Welcome to Cornie's game !!">
+
+        <form>
+          
+          <Grid container 
+              sx={{
+                  '& .MuiOutlinedInput-root': {
+                    '&:hover fieldset': {
+                      borderColor: 'Orchid',
+                    },
+                  },
+                }}>
+            <Grid item xs={12} sx={{mt: 2}}>
+              <TextField 
+                autoFocus
+                label="Email" 
+                type="email" 
+                placeholder="mail@google.com" 
+                fullWidth 
+                name="email"
+                value={formState.email}
+                onChange={onInputChange}
+                error={formValidation.emailValid !== null && formSubmitted}
+                helperText={(formValidation.emailValid && formSubmitted) && formValidation.emailValid}
+              />
+            </Grid>
+            <Grid item xs={12} sx={{mt: 2}}>
+              <TextField 
+                label="Password" 
+                type={showPassword ? 'text' : 'password'}
+                InputProps={{
+                  endAdornment: (
+                      <InputAdornment position="end">
+                        <IconButton
+                          color="primary"
+                          aria-label="toggle password visibility"
+                          onClick={handleClickShowPassword}
+                          onMouseDown={handleMouseDownPassword}
+                          edge="end"
+                        >
+                        {showPassword ? <VisibilityOff /> : <Visibility />}
+                      </IconButton>
+                    </InputAdornment>
+                  )
+                }}
+                placeholder="******" 
+                fullWidth 
+                name="password"
+                value={formState.password}
+                onChange={onInputChange}
+                error={formValidation.passwordValid !== null && formSubmitted}
```

Comment:
Is `formValidation.passwordValid` string or null? Then, maybe let's use a more meaningful name such as `passwordValidationError`